website/tutorial/running-a-kubernetes-cluster.md

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 3
+title: Running a Kubernetes cluster
+description: Running a Kubernetes cluster
+keywords: [podman desktop, podman, Kubernetes]
+tags: [podman-desktop, running-a-kubernetes-cluster, set-up-a-cluster]
+---
+
+# Running a Kubernetes cluster
+
+This tutorial covers the following end-to-end tasks required to run a Kubernetes cluster with the help of an extension:
+
+- Installing the extension
+- Installing CLI on your local machine
+- Applicable on Windows: Configure Podman on WSL
+- Creating and running a Kubernetes cluster
+
+You can use extensions, such as Kind, MiniKube, and others to start a local Kubernetes development cluster. When you have a running Kubernetes cluster, you can easily develop and test Kubernetes applications before deploying them to production. This tutorial focuses on creating a Kubernetes cluster using the Minikube extension.
```

Comment:
avoid "and others" maybe? Make `Kind` as a link to our kind doc maybe?

Reply:
Added an additional resources section in the end.